app/controllers/home.js

```diff
@@ -1,8 +1,19 @@
 import Ember from 'ember';
 
-export default Ember.Controller.extend({
+const defaultOrderBy = 'Title:asc';
 
+export default Ember.Controller.extend({
+  queryParams: ['orderBy', 'skip'],
```

Comment:
Again a lot of work for the component user. All this was encapsulated in my av-table component implementation. We end up repeating this thing in every controller we end up writing in our code. @artzte